src/pudl/transform/eia.py

```diff
@@ -1278,6 +1268,9 @@ def harvested_entity(context, **clean_dfs):
         logger.info(f"Harvesting IDs & consistent static attributes for EIA {entity}")
         eia_settings = context.resources.dataset_settings.eia
         debug = context.op_config["debug"]
+        clean_dfs = {
+            df_name: PUDL_PACKAGE.encode(clean_dfs[df_name]) for df_name in clean_dfs
+        }
```

Comment:
Ensures that all dataframes headed into the harvesting process are encoded, if they weren't in their transforms already.

Reply:
inside of the harvesting process you removed both the encoding and the `apply_pudl_dtypes`. did you intend to add `apply_pudl_dtypes` here?

Reply:
I think I removed it because the tables are immediately written to the DB after harvesting and `enforce_schema` applies dtypes. Actually looking back there now I see that `enforce_schema()` is called within the harvesting function, which might even be superfluous, given that the dataframes are written to the database pretty much immediately after being returned I think?

Reply:
yes good point!